Mathlib/LinearAlgebra/Span/Basic.lean

```diff
@@ -618,10 +618,15 @@ def toSpanNonzeroSingleton : R ≃ₗ[R] R ∙ x :=
     (LinearEquiv.ofEq (range <| toSpanSingleton R M x) (R ∙ x) (span_singleton_eq_range R M x).symm)
 
 @[simp] theorem toSpanNonzeroSingleton_apply (t : R) :
-    LinearEquiv.toSpanNonzeroSingleton R M x h t =
+    toSpanNonzeroSingleton R M x h t =
       (⟨t • x, Submodule.smul_mem _ _ (Submodule.mem_span_singleton_self x)⟩ : R ∙ x) := by
   rfl
 
+@[simp]
+lemma toSpanNonzeroSingleton_symm_apply_smul (m : R ∙ x) :
+    (toSpanNonzeroSingleton R M x h).symm m • x = m :=
```

Comment:
There's a coercion on the RHS, right? Could you write it explicitly for clarity?